protocol-definitions/Jet.yaml

```diff
@@ -416,3 +416,28 @@ methods:
           nullable: false
           since: 2.6
           doc: 'True if processed successfully, false otherwise'
+  - id: 21
+    name: updateJobConfig
+    since: 2.6
+    doc: ''
+    request:
+      retryable: true
```

Comment:
is this really retryable? is it an idempotent operation?

Reply:
Yes, it is idempotent.